Assets/HoloToolkit/Common/Shaders/Standard.shader

```diff
@@ -389,17 +391,9 @@ Shader "MixedRealityToolkit/Standard"
 
                 o.scale.z = minScale;
                 float scaleRatio = min(o.scale.x, o.scale.y) / max(o.scale.x, o.scale.y);
-
-                if (o.scale.x > o.scale.y)
-                {
-                    o.uv.z = 1.0 - (borderWidth * scaleRatio);
-                    o.uv.w = 1.0 - borderWidth;
-                }
-                else
-                {
-                    o.uv.z = 1.0 - borderWidth;
-                    o.uv.w = 1.0 - (borderWidth * scaleRatio);
-                }
+                
+				o.uv.z = IF(o.scale.x > o.scale.y, 1.0 - (borderWidth * scaleRatio), 1.0 - borderWidth);
+				o.uv.w = IF(o.scale.x > o.scale.y, 1.0 - borderWidth, 1.0 - (borderWidth * scaleRatio));
```

Comment:
Formatting is off here. Be sure to check your space/tab settings.

Reply:
I fixed it in an update now.